tests/src/Template/Provider/BaseProviderTest.php

```diff
@@ -51,6 +51,12 @@ final class BaseProviderTest extends Tests\ContainerAwareTestCase
     private Tests\Fixtures\DummyComposerProvider $subject;
     private MockWebServer\MockWebServer $server;
 
+    public function testConstruct(): void
+    {
+        $this->setUp();
+        self::expectNotToPerformAssertions();
+    }
```

Comment:
@eliashaeussler Could you perhaps give this a second look, please. It seems off since we are setting the test up twice here in this Class. I did this to ensure `this->subject` was initialized.